src/Docker/Manager/ContainerManager.php

```diff
@@ -285,6 +285,28 @@ public function execstart($execid, callable $callback = null, $detach = false, $
     }
 
     /**
+     * Get details about an executive
+     *
+     * @param string $execid Identifier from exec()
+     *
+     * @throws \Docker\Exception\UnexpectedStatusCodeException
+     *
+     * @return stdClass
```

Comment:
That's inconsistent with the other methods that returns a `ResponseInterface` object btw :eyes: